README.md

```diff
@@ -0,0 +1,277 @@
+<a name="readme-top"></a>
```

Comment:
- [ ] It's great that you have provided a video presentation and i enjoyed it :clap: i guess you missed the last requirement for your video which was you need to show a piece of code which you are proud of making :+1: Kindly re-record your video and don't forget about this requirement :+1:

Reply:
oh yeah it does, on it. Thanks for notifying me